sql/core/src/test/scala/org/apache/spark/sql/ColumnExpressionSuite.scala

```diff
@@ -129,6 +129,13 @@ class ColumnExpressionSuite extends QueryTest with SharedSQLContext {
       Row(1) :: Row(2) :: Row(3) :: Nil)
   }
 
+  test("single posexplode") {
```

Comment:
Is `sql.GeneratorSuite.scala` okay? The package is different from `expression.GeneratorSuite`.